### DISCUSSION TITLE
[@types/esri-leaflet] missing vectorBasemapLayer

### DISCUSSION THREAD

**Discussion by davetapley** [DefinitelyTyped/DefinitelyTyped#66456] [Issues with a @types package]
214b1062b7779e3d9964cb7c9b6d954e5a6b97c8 introduced ⬇️ but there is no `vectorBasemapLayer`
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b569f03147c4526b9d2b5ca5b701ac7925d388ef/types/esri-leaflet/index.d.ts#L80

cc @murphyryan1 unless I'm missing something? 🤔

[a comment by an automated account is not shown]

**Comment by murphyryan1** [comment]
> [214b106](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/214b1062b7779e3d9964cb7c9b6d954e5a6b97c8) introduced ⬇️ but there is no `vectorBasemapLayer`
> 
> https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b569f03147c4526b9d2b5ca5b701ac7925d388ef/types/esri-leaflet/index.d.ts#L80
> 
> cc @murphyryan1 unless I'm missing something? 🤔

Its a plugin I believe: https://developers.arcgis.com/esri-leaflet/api-reference/layers/vector-basemap/
